examples/models/llama2/export_llama_lib.py

```diff
@@ -143,6 +144,79 @@ def replace_sdpa_with_custom_op(module: torch.nn.Module) -> torch.nn.Module:
     return module
 
 
+class SDPASimple(torch.nn.Module):
```

Comment:
Can we just keep sdpa as is by registering it as a custom qnn op that qnn delegate can directly consume?

Reply:
It's not just for qnn. Basically for any backend that doesn't support sdpa, it makes more sense to use this version instead.